lib/app.rb

```diff
@@ -41,10 +41,12 @@ def run
     print 'Option: '
     op = gets.chomp.to_i
 
-    menu_options.map do |option|
-      instance_eval(option[:method]) if op == option[:id]
+    if op.positive? && op < 14
```

Comment:
in this line, can we have ```options.length```? I think it would be easier to understand than using a magic number ```14```

Reply:
@Trictonicmp Thanks for the review, I updated it. 😄